packages/react/src/ArqueroDetailsList/types.ts

```diff
@@ -2,15 +2,24 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import { IColumn, IDetailsListProps } from '@fluentui/react'
+import { IColumn, IDetailsListProps, IDropdownOption } from '@fluentui/react'
 import ColumnTable from 'arquero/dist/types/table/column-table'
 
 export type ColumnRenderFunction = (
 	item?: any,
 	index?: number,
 	column?: IColumn,
+	arrayAsDropdown?: boolean,
 ) => any
 
+export type DropdownOptionSelect =
```

Comment:
I think this select handler would be more useful if we could pass along more information about the cell. A normal dropdown is just a list of items to choose, but in this case you might want to know what column and row it was as well. IDropdownOption (extends ISelectableOption) has a generic type T and a `data` field to hold whatever you'd like - that could be a way to do this by passing through the column object and rowIndex.